website/registrations/templates/registrations/email/renewal_accepted.txt

```diff
@@ -5,12 +5,17 @@ We have some wonderful news for you: your membership renewal has been accepted!
 Unfortunately there is still one little thing that we have to take care of before
 you get access to everything Thalia has to offer once again: paying your membership fees.
 
-The membership fees are €{{ fees }} and can be paid using cash, a debit card or via wire transfer.
-You can transfer the amount of €{{ fees }} to NL 23 INGB 0006 2341 84 with the description "Contribution {{ name }}".
+The membership fees are €{{ fees }}. Those can be paid in a few ways:
+
+1. You can let us withdraw the money from your bank account via Thalia Pay. For this, you
+will need to have a bank account added to your account and have signed a direct debit mandate.
+To pay with Thalia Pay, go to https://thalia.nu/user/membership/.
```

Comment:
You should be able to use the base url instead of using the thalia.nu domain itself.